website/cue/reference/components/sinks/base/aws_s3.cue

```diff
@@ -885,6 +885,17 @@ base: components: sinks: aws_s3: configuration: {
 			}
 		}
 	}
+	timezone: {
+		description: """
+			Timezone reference.
+
+			This can refer to any valid timezone as defined in the [TZ database][tzdb], or "local" which refers to the system local timezone.
+
+			[tzdb]: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
```

Comment:
```suggestion
			Timezone to use for any date specifiers in template strings.

			This can refer to any valid timezone as defined in the [TZ database][tzdb], or "local" which refers to the system local timezone. It will default to the [globally configured timezone](https://vector.dev/docs/reference/configuration/global-options/#timezone).

			[tzdb]: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
```

Reply:
(this change will need to be on the config structs)

Reply:
> (this change will need to be on the config structs)

I think this was generated when I ran `make check-component-docs`.